server/routes/hooks.js

```diff
@@ -3,22 +3,81 @@ import { middlewares } from 'auth0-extension-express-tools';
 
 import config from '../lib/config';
 import logger from '../lib/logger';
+import { getResourceServer, createResourceServer, deleteResourceServer } from '../lib/queries';
 
 export default () => {
   const hooks = router();
   const hookValidator = middlewares
     .validateHookToken(config('AUTH0_DOMAIN'), config('WT_URL'), config('EXTENSION_SECRET'));
 
```

Comment:
As part of install/update we need to deploy a rule that checks if you request `manage:applications`. If you are, we should block you from logging in.

This can be used as a base for it: https://github.com/auth0-extensions/auth0-box-platform-extension/blob/master/server/lib/rules/box-rule.js#L5-L8

On uninstall it should be delete